src/chrome/content/rules/idnes-cz.xml

```diff
@@ -0,0 +1,39 @@
+<ruleset name="iDNES.cz">
+	<target host="aplikace.idnes.cz" />
+	<target host="auto.idnes.cz" />
+	<target host="blog.idnes.cz" />
+	<target host="bonusweb.idnes.cz" />
+	<target host="bydleni.idnes.cz" />
+	<target host="cestovani.idnes.cz" />
+	<target host="doprava.idnes.cz" />
+	<target host="ekonomika.idnes.cz" />
+	<target host="finance.idnes.cz" />
+	<target host="hobby.idnes.cz" />
+	<target host="icka.idnes.cz" />
+	<target host="kraje.idnes.cz" />
+	<target host="mobil.idnes.cz" />
+	<target host="muj.idnes.cz" />
+	<target host="ona.idnes.cz" />
+	<target host="pocasi.idnes.cz" />
+	<target host="rajce.idnes.cz" />
+	<target host="*.rajce.idnes.cz" />
+	<target host="revue.idnes.cz" />
+	<target host="rungo.idnes.cz" />
+	<target host="sdeleni.idnes.cz" />
+	<target host="servis.idnes.cz" />
+	<target host="sport.idnes.cz" />
+	<target host="technet.idnes.cz" />
+	<target host="tvprogram.idnes.cz" />
+	<target host="video.idnes.cz" />
+	<target host="wiki.idnes.cz" />
+	<target host="zpravy.idnes.cz" />
```

Comment:
I found some more:

idnes.cz
www.idnes.cz
alik.idnes.cz
blog.idnes.cz
brno.idnes.cz
budejovice.idnes.cz
cestiny.idnes.cz
data.idnes.cz
fotbal.idnes.cz
hledej.idnes.cz
hokej.idnes.cz
jihlava.idnes.cz
kultura.zpravy.idnes.cz
liberec.idnes.cz
mobilnihry.idnes.cz
olomouc.idnes.cz
onlinehry.idnes.cz
ostrava.idnes.cz
pardubice.idnes.cz
plzen.idnes.cz
praha.idnes.cz
r.idnes.cz
reality.idnes.cz
rss.idnes.cz
servix.idnes.cz
topkontakt.idnes.cz
usti.idnes.cz
vary.idnes.cz
vice.idnes.cz
volby.idnes.cz
xman.idnes.cz
zlin.idnes.cz
kultura.zpravy.idnes.cz


Please add non-working ones as comments on the top, to document the issue.